redaxo/src/core/lang/de_de.lang

```diff
@@ -258,8 +258,8 @@ setup_700 = Setup: Schritt 7 von 7 / Heureka
 setup_701 = Leider konnte die config.yml nicht beschrieben werden. Bitte ändern Sie die Datei manuell über FTP.
 setup_702 = Herzlichen Glückwunsch zur REDAXO-Installation!
 setup_703 = Bitte noch dieses beachten:
-setup_704 = <ol><li>Mit dem {0}eingerichtetem Zugang einloggen{1}</li><li>Importieren einer Demo über [wenn eingeloggt]: Import/Export und dort jeweils die Datenbank und Files installieren.</li></ol>
-setup_705 = Viel Spass und Erfolg<br />Das REDAXO Team
+setup_704 = <ol><li>Mit dem {0}eingerichtetem Zugang einloggen{1}</li><li>Innerhalb der Community haben wir mehrere Website-Demos entwickelt, die typische Anwendungsfälle mit REDAXO zeigen. Sie können nach Login über das bereits enthaltene <strong>Install</strong>-AddOn runtergeladen werden. <a href="https://redaxo.org/download/website-demos/">Infos zu Website-Demos auf redaxo.org</a></li></ol>
```

Comment:
Wobei ich die Link-Tags glaube ich rausziehen würde, so wie vorne im Text. Damit es einheitlich ist.
Und eigentlich finde ich auch nicht gut, dass wir hierfür `rawMsg` verwenden, und ohne könnte man den Link hinten gar nicht so platzieren.

Reply:
Aber dann wäre die URL im Code, findest du das sinnvoll? Für mein Bauchgefühl sind externe URLs besser in den Sprachdateien aufgehoben.

Reply:
Weiß ich nicht so recht.
Mir gefällt vor allem das `rawMsg` nicht (was wir aber schon vor dem PR hatten), weil man dann plötzlich aufpassen muss, alles korrekt zu escapen. Also `&amp;` statt `&`  etc.

Aber dann lass es ruhig so. Verstehe auch deinen Punkt, und wie gesagt, das `rawMsg` hatten wir ja sowieso schon.